name-computer/map.jinja

```diff
@@ -23,6 +26,13 @@ RedHat:
       - hostname: {{ name }}
       - apply_hostname: True
       - retain_settings: True
+
+  Update /etc/hosts:
+    host.present:
+      - ip: {{ salt['network.interface'](salt['network.default_route']('inet')[0]['interface'])[0]['address'] }}
+      - names:
+        - {{ name|lower }}.{{ dns_domain|lower }}
```

Comment:
I think this entry should be conditional on `dns_name`... If empty, don't set it

Reply:
Yeah... Was gonna address that while I had things open for the first comment. =)